### PR TITLE
Fallback to config LiveKit URL if there is no URL coming from the js-sdk

### PR DESCRIPTION
**Pull request by SimonBrandner**

### CODE REVIEW

**Comment by toger5** [comment]
That was quick (give me a sec to review)

File: src/room/GroupCallView.tsx [hunk 1]
```diff
@@ -219,7 +220,9 @@ export function GroupCallView({
     undefined
   );
 
-  const livekitServiceURL = groupCall.foci[0]?.livekitServiceUrl;
+  const livekitServiceURL =
+    groupCall.foci[0]?.livekitServiceUrl ??
+    Config.get().livekit.livekit_service_url;
```
Comment (toger5) [inline src/room/GroupCallView.tsx:225]: I guess I am assuming correctly, that a failed network request or an unset value will result in foci[0] to be undefined?

**Review by toger5 — APPROVED** [review]
Cool we have this in

File: src/room/GroupCallView.tsx (hunk 1 above)
Comment (SimonBrandner) [reply src/room/GroupCallView.tsx:225]: That's actually the following step. This only falls back when the value is undefined (i.e. the `m.call` event doesn't have the field). After we get the URL here we pass it over to the `OpenIDLoader` which actually does the request

**Review by toger5 — APPROVED** [review]

**Pull request merged**